backend/tests/clubs/test_ticketing.py

```diff
@@ -58,10 +58,16 @@ def commonSetUp(self):
     )
 
     self.ticket_totals = [
+        {"type": "free", "count": 30, "price": 0.0},
         {"type": "normal", "count": 20, "price": 15.0},
         {"type": "premium", "count": 10, "price": 30.0},
     ]
 
+    self.tickets0 = [
+        Ticket.objects.create(type="free", event=self.event1, price=0.0)
+        for _ in range(30)
+    ]
+
```

Comment:
nit: these will only be useful for the `initiate_checkout` test case for free tickets, would prefer not to add them globally.

Reply:
refer to previous reply